spt/features/collision_group.cpp

```diff
@@ -57,7 +67,25 @@ CON_COMMAND_F(y_spt_set_collision_group, "Set player's collision group\nUsually:
 void CollisionGroup::LoadFeature()
 {
 	if (ORIG_SetCollisionGroup)
+	{
 		InitCommand(y_spt_set_collision_group);
+		SptImGuiGroup::Cheats_PlayerCollisionGroup.RegisterUserCallback(
+		    []()
+		    {
+			    static long long val = 5;
+			    SptImGui::InputTextInteger("value", "enter integer", val, 10);
+			    ImGui::SameLine();
+			    auto playerPtr = utils::GetServerPlayer();
+			    ImGui::BeginDisabled(!playerPtr);
+			    if (ImGui::Button("set"))
+				    spt_collisiongroup.ORIG_SetCollisionGroup(playerPtr, (int)val);
+			    ImGui::EndDisabled();
+			    if (!playerPtr)
+				    ImGui::SetItemTooltip("server player not available");
+			    ImGui::SameLine();
+			    SptImGui::CmdHelpMarkerWithName(y_spt_set_collision_group_command);
+		    });
+	}
```

Comment:
would this be better with a dropdown with the names of the collision groups? more effort though